src/styles/main.scss

```diff
@@ -1 +1,62 @@
-// put code here
+@import "./utils/general.scss";
+@import "./utils/varibales.scss";
+
+.watch {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border: 10px dashed #000;
+  border-radius: $watch-radius;
+  margin: 0 auto;
+  width: $wach-size;
```

Comment:
create please separated file and move all these blocks there, like `stopwatch.scss`